test/standalone/test-index.js

```diff
@@ -121,4 +122,43 @@ describe('index.js', function() {
     assert.equal(typeof agent.addTransactionLabel, 'function');
   });
 
+  it('should allow start and end span calls when disabled', function() {
+    agent.stop();
+    var span = agent.startSpan();
```

Comment:
Not a comment on your change, but should we stop returning a value from startSpan?